applications/ssotap/secrets-idfint.yaml

```diff
@@ -0,0 +1,3 @@
+uws-db-password:
+  description: >-
+    Password to external UWS PostgreSQL server
```

Comment:
"Password for" would sound a bit better to my ears, here and elsewhere.

Reply:
Yep sounds right, I've changes it to match your suggestion